code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -333,16 +325,15 @@
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
 /datum/reagent/teslium/energized_jelly/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
-	. = ..()
-	if(isjellyperson(affected_mob))
-		shock_timer = 0 //immune to shocks
-		affected_mob.AdjustAllImmobility(-40  *REM * seconds_per_tick)
-		if(affected_mob.adjustStaminaLoss(-2 * REM * seconds_per_tick, updating_stamina = FALSE))
-			. = UPDATE_MOB_HEALTH
-		if(is_species(affected_mob, /datum/species/jelly/luminescent))
-			var/mob/living/carbon/human/affected_human = affected_mob
-			var/datum/species/jelly/luminescent/slime_species = affected_human.dna.species
-			slime_species.extract_cooldown = max(slime_species.extract_cooldown - (2 SECONDS * REM * seconds_per_tick), 0)
+	if(!isjellyperson(affected_mob)) //jellypeople don't get shocked.
```

Comment:
This comment makes it seem like it's early returning to prevent jellypeople from getting shocked but it's kinda inaccurate, maybe this is better

```suggestion
	if(!isjellyperson(affected_mob)) //everyone but jellypeople get shocked as normal.
```